data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_ability.lst

```diff
@@ -0,0 +1,88 @@
+Absorb Magic	KEY:Absorb Magic	TYPE:Supernatural.SpecialQuality	CATEGORY:Special Ability			DESC:Gain HP equal to the damage that damaging magic would do, except for fire and cold magic (which is reduced) and which dissipated after a round.  Nondamaging spells gratn 1HP per spell level.  Mind-control spells have no effect whatsoever.  Can drain 1d4 charges from a magical item on touch, and for every other round of contact thereafter.  Can negate the effect of non-charged items for 1d4 rounds by touch. 
```

Comment:
This entire file is outside of current conventions. Each race granted ability should follow the KEY:RaceName ~ AbilityName convention. This lacks or follows older outdated conventions.